From: William Grzybowski <william@grzy.org>
Date: Fri, 26 Jul 2019 08:43:03 -0300
Subject: Do not install pydevd_attach_to_process

---
 MANIFEST.in |  6 +++---
 setup.py    | 16 ----------------
 2 files changed, 3 insertions(+), 19 deletions(-)

diff --git a/MANIFEST.in b/MANIFEST.in
index bf7ead1..395351a 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,4 +1,4 @@
-include *.rst *.txt *.md LICENSE .travis.yml appveyor.yml *.pyx
-recursive-include pydevd_attach_to_process *.py *.dll *.so *.dylib *.txt *.c *.h *.bat Makefile *.sh *.pyx
+include *.rst *.txt *.md LICENSE
+recursive-include pydevd_attach_to_process *.py *.so *.txt Makefile *.sh
 recursive-include _pydevd_bundle *.pyx
-recursive-include build_tools *.py
\ No newline at end of file
+recursive-include build_tools *.py
diff --git a/setup.py b/setup.py
index 049813e..378cb1a 100644
--- a/setup.py
+++ b/setup.py
@@ -31,21 +31,6 @@ class BinaryDistribution(Distribution):
     def is_pure(self):
         return False
 
-data_files = []
-
-def accept_file(f):
-    f = f.lower()
-    for ext in '.py .dll .so .dylib .txt .cpp .h .bat .c .sh .md .txt'.split():
-        if f.endswith(ext):
-            return True
-
-    return f in ['readme', 'makefile']
-
-data_files.append(('pydevd_attach_to_process', [os.path.join('pydevd_attach_to_process', f) for f in os.listdir('pydevd_attach_to_process') if accept_file(f)]))
-for root, dirs, files in os.walk("pydevd_attach_to_process"):
-    for d in dirs:
-        data_files.append((os.path.join(root, d), [os.path.join(root, d, f) for f in os.listdir(os.path.join(root, d)) if accept_file(f)]))
-
 import pydevd
 version = pydevd.__version__
 
@@ -113,7 +98,6 @@ args = dict(
             'pydevd = pydevd:main',
         ],
     },
-    data_files=data_files,
     keywords=['pydev', 'pydevd', 'pydev.debugger'],
     include_package_data=True,
     zip_safe=False,
